### DISCUSSION TITLE
set default path for sftp connections

### DISCUSSION THREAD

**Discussion by mikekazakov** [mikekazakov/nimble-commander#249] [Ideas]
FTP supports this, it would be useful for SFTP to do that too.

**Comment by mikekazakov** [comment]
Also, regarding naming:
It is not obvious in the SFTP connection dialogue, that the password field would used to decrypt the key. Perhaps, setting the field placeholder to "Server password or key password" would help?

**Comment by mikekazakov** [comment]
Also, it's possible to check whether the key is available and can be decrypted right inside this dialog, reducing an amount of possible misunderstandings.

**Comment by varrav** [comment]
It is very necessary that in the SFTP it was possible to specify the initial folder
